doc/sphinx/developer_manual/contributing_to_the_documentation/building_documentation.md

```diff
@@ -0,0 +1,29 @@
+(part:dev_manual:chap:contribute_to_doc:sec:building_doc)=
+Building documentation
+
+To build the documentation locally, you first need to ensure you have the right python packages installed. The list of packages required to build the documentation is located in `$WORLD_BUILDER_SOURCE_DIR/doc/requirements.txt`, and can be installed using 
+
+:::{code-block}
+pip install -r $WORLD_BUILDER_SOURCE_DIR/doc/requirements.txt \\
+:::
+
+Now that the python packages are installed, we need to toggle the option in cmake to build the target for the GWB documention, this can be done using `ccmake` and enabling `WB_BUILD_DOCUMENTATION=ON`. Run the following command, and use the interactive GUI to do this:
+
+:::{code-block}
+cd $WORLD_BUILDER_SOURCE_DIR/build \\
+ccmake .
+:::
+Now we compile the GWB normally by running the following:
+
+:::{code-block}
+cmake .. && make
```

Comment:
This step should not be necessary, right?

Reply:
Yes you're right!